gatsby-site/src/components/Contact/contact.css

```diff
@@ -0,0 +1,85 @@
+:root {
+    --colarkfontcolor: #0111FC;
+    --colarkfont: 'Mukta Malar', sans-serif;
+    --background: #FFFFFF;
+}
+
+.form-container {
+    width: 100%;
+    text-align: center;
+    padding: 40px;
+}
+
+.left-image {
+    width: 50%;
+    margin: 20px;
+}
+
+.right-form {
+    width: 40%;
+    word-break: keep-all;
+}
+
+.right-form, .left-image{
+    display: inline-block;
+    padding: 50px;
+    align-self: center;
+}
+
+.submitbtn {
+    float: right;
+    width: 40%;
+    padding: 0;
+    font-family: var(--colarkfont);
+    margin-top: 25px;
+    background-color: #FF6969;
+    border: 1px solid #FF6969;
+}
+
+.form-title {
+    font-family: var(--colarkfont);
+    color: var(--colarkfontcolor);
+    font-weight: 700;
+    margin-bottom: 30px;
+    text-align: left;
+}
+
+#exampleText, #email, #name {
```

Comment:
instead of styling IDs, does the reactstrap Input component support passing in a class prop? you could define one here and pass the class string when you use the Input component